src/modules/ToDoList.js

```diff
@@ -34,13 +34,12 @@ export default class ToDoList {
   }
 
   removeCompleted() {
-    const arr = [];
-    for (let i = 0; i < this.taskList.length; i += 1) {
-      if (!this.taskList[i].isCompleted()) {
-        arr.push(this.taskList[i]);
+    const len = this.taskList.length;
+    for (let i = len - 1; i >= 0; i -= 1) {
+      if (this.taskList[i].completed) {
+        this.removeTask(i);
       }
     }
```

Comment:
- [ ] Kindly refactor your clear completed functionality to use the `filter()` method.